apps/service-portal/src/assets/images/bus.svg

```diff
@@ -1,433 +1 @@
-<svg width="432" height="445" viewBox="0 0 432 445" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This file seems to be still in use in the service-portal. 😱

Reply:
Seems like the file changed from 433 lines to 1 line (but we didnt change anything), but I have reverted the changes in this file